website/content/api-docs/system/internal-counters.mdx

```diff
@@ -239,29 +223,19 @@ Vault lead to the new clients for each month.
 }
 ```
 
-- The `distinct_entities` field name has been deprecated since Vault 1.10. Refer to
-`entity_clients` field instead. The `distinct_entities` field is currently
-returned by the API for backward compatibility and it may be removed in the future.
+- The response includes month data for the entire queried period, but may
```

Comment:
This addresses the confusion in this [thread](https://hashicorp.slack.com/archives/C0282NFTX4P/p1721231854554999)